docs/docs/composition-vs-inheritance.md

```diff
@@ -44,7 +44,7 @@ function WelcomeDialog() {
 }
 ```
 
-[Try it on CodePen.](http://codepen.io/gaearon/pen/ozqNOV?editors=0010)
+[Try it on CodePen.](https://codepen.io/gaearon/pen/ozqNOV?editors=0010)
```

Comment:
Good call switching these all over to your account. Maybe we could create a shared React Codepen account so that anyone can make changes without updating the link? Unless it's too much hassle to switch between accounts.

Reply:
That would be great.